cogs/badges.py

```diff
@@ -153,16 +153,16 @@ def get_user_badges(self, discord_id):
                 userBadges["achievements"].append(self.rockstarBadge)
             if discordMemberData[0]["has_introduced"]:
                 userBadges["achievements"].append(self.apprenticeBadge)
-        contributorData = SupabaseClient().read(
+        contributorData = PostgresClient().read(
             "contributors_registration", query_key="discord_id", query_value=discord_id
         )
         if contributorData:
             github_id = contributorData[0]["github_id"]
             prData = {
-                "raised": SupabaseClient().read(
+                "raised": PostgresClient().read(
                     table="connected_prs", query_key="raised_by", query_value=github_id
                 ),
-                "merged": SupabaseClient(table="connected_prs").read(
+                "merged": PostgresClient(table="connected_prs").read(
```

Comment:
This seems to be a different syntax. We should have a uniform syntax across the board.